src/panes/Files.js

```diff
@@ -144,6 +177,7 @@ class Files extends Component {
             <IconButton active={this.state.sticky} onClick={this.toggleStickWindow} icon='eye' />
 
             <div className='right'>
+              <IconButton onClick={this.addFromIPFS} icon='pin-alt' />
```

Comment:
`IconButton` should accept `title` as one of its props, and use it like this:
```html
<button title="Add by IPFS Path">[icon]</button>
```

We really need a tooltip that describes what the button does :)